### ISSUE TITLE
WIP: Frontend Platform :: Refresh Prep Tracking issue

### ISSUE THREAD

**Issue by pdubroy** [sourcegraph/sourcegraph#19555]
### Plan

This issue tracks work by the Frontend Platform team in preparation for the upcoming design refresh.

### Tracked issues

- [ ] [Ability to run e2e and regression tests locally](https://github.com/sourcegraph/sourcegraph/issues/19496)
- [ ] [Review and add visual regression tests for design refresh](https://github.com/sourcegraph/sourcegraph/issues/19520)
- [x] Finish the [high-level plan](https://docs.google.com/document/d/1Zomc7cb2uWtp86Cvl6_W0r2aaF-vdGREVeCHVOZNMOU/edit#) and get feedback from @frontend-devs
- [x] [Add developer docs about Wildcard design system](https://github.com/sourcegraph/sourcegraph/issues/19351)
- [x] [CSS modularity for wildcard components](https://github.com/sourcegraph/sourcegraph/issues/19349)
- [x] [Standardise an approach for supporting style-agnostic components](https://github.com/sourcegraph/sourcegraph/issues/19140)
- [x] Summarize accessibility strategy and tooling: [Frontend Platform - Wildcard Accessibility](https://docs.google.com/document/d/1MuRRlxIYy8QYOCMXCr2PoO4e3WcdKsZoiAcS57k0U_A/edit#)

**Comment by pdubroy** [comment]
Closing this since we are in the execution phase of the refresh now. Comments on the two unfinished items:

- Review and add visual regression tests for design refresh => we made progress here and will continue to improve coverage as needed.
- Ability to run e2e and regression tests locally => Did not make progress, but we will tackle this if needed during the refresh, rather than working on it proactively.
